book/website/community-handbook/translation/translation-workflow.md

```diff
@@ -4,13 +4,40 @@
 We are members of _The Turing Way_ community with different motivations to localise the content to different languages.
 Feel free to join the `#translation` channel in [_The Turing Way_ slack](https://theturingway.slack.com).
 You will also have to create a Crowdin account but more on that {ref}`in the next section<ch-translation-localisation-journey>`
-The team has a fortnightly call on Tuesdays at 5pm UTC to co-work, discuss, and check the progress of the translation.
-These sessions are open to anyone interested in participating.
 
 You can find these activities in [_The Turing Way_ calendar](https://calendar.google.com/calendar?cid=dGhldHVyaW5nd2F5QGdtYWlsLmNvbQ).
 Please note that these calls are in English but we are happy to facilitate other co-working calls in different languages.
 The deployment of these translated versions is work in progress, and some parts of this workflow may change in the future. Feel free to contact anyone on the `#translation` channel of the _The Turing Way_ Slack if you have any ideas and feedback to help our work.
 
+## Structure of the Meetings
+### Co-working Calls & Onboarding Calls
+- **Frequency**: Once every month
+- **Chair**: Selected from leads
+- **Membership**: Open to anyone interested in localisation
+- **Objectives**:
+  - A facilitator assists with onboarding in the breakout room.
+  - It is an 'Ask Me Anything' call for any questions.
+  - The main room is dedicated to working on milestones for the localisation work.
+  - Tasks are prepared in advance (in Crowdin Kanban) to support new members.
+### Community Meeting
+- **Frequency**: This is integrated into the TTW Collaboration Café. 
+- **Chair**: Selected from leads
+- **Membership**: Open to anyone interested in localisation.
+- **Objectives**: A forum to discuss deliverables/outputs (e.g. toolkits, glossary, blog post, ...etc), consult the community, and seek feedback.
```

Comment:
```suggestion
- **Objectives**: A forum to discuss deliverables/outputs (for example: toolkits, glossary, blog posts), consult the community, and seek feedback.
```

Reply:
Nice catch - I've done the same, and it breaks our compiler I believe?